articles/openshift/oshift-how-v4-update.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Supported customer update strategy for version 4 clusters
+description: Provides information on supported strategies for self-service customer updates
+services: openshift
+author: Kieran O'Neill
+reviewer: 
+lastreviewed: 24/07/2020
+toc_rootlink: Reference
+toc_sub1: v4
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Supported customer update strategy for version 4 clusters
+toc_fullpath: Reference/oshift-how-v4-update.md
+toc_mdlink: oshift-how-v4-update.md
+---
+
+# Supported customer update strategy for version 4 clusters
+
+## Update Information
+
+In OpenShift version 4, updates of the cluster and operating system of the nodes have been exposed to the customer through the cluster. You can update clusters through the UI and the cluster will then deal with updating the relevant operator components and the OS version of the RHCOS nodes running as masters, infras and workers. The OS version of the nodes is tied to the version of the cluster and is thoroughly tested by Red Hat pipelines prior to release.
+
+## Supported Strategy
+
+There are three update channels available to choose from:
+
+- **Stable** - This is the recommended channel for production clusters. Updates are released to this channel after data has been collected by Red Hat from customers running updates from the fast channel and stability of the updates has been confirmed.
+
+- **Fast** - This channel is fully supported. Updates are released to this channel after performing well in the candidate channel.
+
+- **Candidate** - This is an unsupported channel that will provide release candidate z stream releases. Do not update via this channel or both UKCloud and Red Hat will be unable to support the cluster.
+
+> [!NOTE]
+> UKCloud asks that all customers remain on the stable channel for production clusters. We recommend using the fast channel only if you require specific bug fixes or security patches.
+
+## Upgrading your cluster
+
+### Prerequisites
+
+- You must be a cluster admin user
+
+- You must contact UKCloud Support and ask for an etcd backup to be taken to ensure there is a restore point in case of corruption during the update
```

Comment:
Is this still accurate? Didn't we discover the clusters auto-backup now prior to an upgrade?

Reply:
It's no longer accurate. This line has been removed. I have also added a mandatory update section, please review.